terracumber_config/tf_files/SUSEManager-4.3-build-validation.tf

```
@@ -248,7 +248,7 @@ module "base_debian" {
   name_prefix = "suma-bv-43-"
   use_avahi   = false
   domain      = "mgr.prv.suse.net"
-  images      = [ "ubuntu1804o", "ubuntu2004o", "ubuntu2204o", "debian10o", "debian11o" ]
+  images      = [ "ubuntu1804o", "ubuntu2204o", "ubuntu2204o", "debian10o", "debian11o" ]
```

you have it duplicated here
```suggestion
  images      = [ "ubuntu1804o", "ubuntu2204o", "debian10o", "debian11o" ]
```

---

Oups! Right! Too fast search-replace :)